### DISCUSSION TITLE
Any way to enable caching for a service with docker compose labels?

### DISCUSSION THREAD

**Discussion by jltjarvinen** [evertramos/nginx-proxy-automation#392] [General]
I would like to play around with caching but can't deduct what labels should be used to get it mapped to the nginx configuration. Any idea how to get it working?

```
# docker-compose.yml
services:
  web1:
    labels:
      - "com...cache=true"
```

**Comment by evertramos** [comment]
Hello @jltjarvinen, not sure how to set it up. You might want to post it on 

https://github.com/nginx-proxy/nginx-proxy/discussions
